source/install/troubleshooting.rst

```diff
@@ -539,6 +539,13 @@ Hitting an Error "Command with a trigger of failed" When Configuring Giphy Integ
 
 When trying to configure the Giphy integration in Mattermost, you may hit the error "Command with a trigger of <keyword> failed". To solve this, you need to edit your ``config.json`` and configure ``AllowedUntrustedInternalConnections`` to contain the hostname of the webhook.
 
+(Gfycat) What to do if gifs are not loading even though they show up in the emoji picker?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Check if the server has access to the gfycat servers if it's behind a proxy or firewall blocking outgoing connections.
```

Comment:
```suggestion
1. Check whether the server has access to the Gfycat servers. It may be behind a proxy or firewall which is blocking outgoing connections.
```

Reply:
How does one check access and, if there is a proxy/firewall, how do you remediate that?